targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -141,6 +141,14 @@ public void OnApplicationFocus(bool isFocused)
             eventsRequests.Enqueue(eventInfo);
 
             initialFocus = false;
+
+            if(!isFocused)
+            {
+                // Force the eventsRequests queue to empty.
+                // If we are losing focus we should make an attempt to push out a focus lost event ASAP
+                Send();
```

Comment:
You might not be sending the message you just Enqueued. You might be sending some earlier queued message.
Though, as I think about it, this might just be a general problem with this batching implementation.
I think this checkin is fine, but we're going to have to do a proper design pass on a real "batching" plugin for pluginmanager, and think carefully through our goals.